### PR TITLE
Updated push notifications troubleshooting

### PR DESCRIPTION
**Pull request by jdiehl**
Added ngrok link (added this because deploying to sandbox will not work)
Explain that a licensed node is needed

@ConnorLand

### CODE REVIEW

**Comment by jdiehl** [comment]
@allard-mx plese review

File: content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ If you did not receive the message, check the application's logs to see if the m
...
 | Error sending message: **Error reading credentials from stream, 'type' field not specified. at PushNotifications.SendFCMMessages (JavaAction : 'GetFCMAccessToken')**. | The wrong private key file was uploaded. | Upload the correct file or [generate a new private key](/refguide/mobile/using-mobile-capabilities/push-notifications/setting-up-google-firebase-cloud-messaging-server/#setting-up-a-service-account) in Firebase and upload it. |
 | Not receiving push notifications on a PWA | Push notifications on PWAs are only shown when the app is in the background or closed | Close the app and try again |
-| Registration fails on a PWA on iOS | To receive push notifications on iOS the app must be served over https and added to the home page | Deploy your app and add it to the home page via the Share button in Safari and try again |
+| Registration fails on a PWA on iOS | To receive push notifications on iOS the app must be served over https and added to the home page | Deploy your app (or use a tool such as [ngrok](https://ngrok.com)), add your app to the home page via the Share button in Safari and try again |
```
Comment (allard-mx) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md]: Can we change `home page` to `home screen` here? As far as I know, `Add to Home Screen` is the official name for this action/option in Safari

Maybe something like `Tap the 'Share' button in Safari, choose the 'Add to Home Screen' option and try again`

File: content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ If you did not receive the message, check the application's logs to see if the m
...
 | Clicking on a push notification does not open the PWA | Push notifications for PWAs require a Web link to be set. | Make sure the Web link is set to a valid path of your application (for example "/") |
 | Application no longer starts after changing index.html | Firebase is not found | Ensure that the script tags that load Firebase are placed above the script tag that loads mxui.js |
+| Messages are not sent (Status remains Pending) | The Push notifications connector relies on scheduled events to function, which are not supported in a free app running | Run your app on a licensed node |
```
Comment (allard-mx) [inline content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md]: Enabling the scheduled event should not be strictly required as far as I know, without it the module should still be able to send out push notifications (albeit to a limited extent). The main purpose for the scheduled event is to automatically retry failed messages. 

All the `CreateAndSendMessageTo*` actions call the `SendMessages` microflow in a task queue. Only if the message fails (and based on the exception), the scheduled event ensures an automatic retry.

**Review by allard-mx — CHANGES_REQUESTED** [review]

File: content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md (hunk 1 above)
Comment (ConnorLand) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md]: Done.

File: content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md (hunk 2 above)
Comment (allard-mx) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md]: @jdiehl Any thoughts? Do you have some more background/context on this note?

Comment (jdiehl) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md]: I have changed the wording. We did receive some comments from customers trying this out in the sandbox and failing to get it working (messages stay in status "Pending")

Comment (allard-mx) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md]: Are you sure that this is related to the scheduled event though? If not, I think we should first research what the root cause of the issue could be when using the module in a sandbox environment

Comment (jdiehl) [reply content/en/docs/refguide/mobile/using-mobile-capabilities/push-notifications/notif-send-test.md]: I have no way to replicate the issue. Let's see if others report it and hold back any changes for now.

**Pull request closed without merging**